packages/manager/apps/pci-gateway/src/pages/edit/EditGateway.page.tsx

```diff
@@ -115,6 +116,7 @@ export default function EditGatewayPage(): JSX.Element {
 
   return (
     <>
+      <HidePreloader />
```

Comment:
HidePreloader was originally on Layout page, the issue is that due to onboarding redirection logic that need to fetch services beforehand to either display onboarding or listing page, there is some delay before the listing page is displayed.
This causes a blank page to be visible because layout is already visible during the service loading.
On the other hand the creation / order page doesn't need the calls to services so the loading time will also be different.

I found the easiest solution in such case would be to hide preloader only when the page is actually visible. I think we shouldn't have the issue if there were no onboarding redirection because in that case we would be able to display the listing page progressively.